pandas/core/frame.py

```diff
@@ -6485,70 +6487,67 @@ def join(self, other, on=None, how='left', lsuffix='', rsuffix='',
 
         Examples
         --------
+        >>> import pandas as pd
+
         >>> caller = pd.DataFrame({'key': ['K0', 'K1', 'K2', 'K3', 'K4', 'K5'],
```

Comment:
do you mind replacing the name `caller` by `df`? we use this convention everywhere, I think it makes sense here too.